composer.json

```diff
@@ -31,7 +31,8 @@
     "league/uri": "^5.2",
     "imgix/imgix-php": "^2.1",
     "verbb/super-table": "^2.0",
-    "ether/tags": "^1.0"
+    "ether/tags": "^1.0",
+    "doublesecretagency/craft-inventory": "^2.0"
```

Comment:
Adding this plugin to our installed list as it's useful to help us migrate towards some of these newer generic field